packages/sdk/src/util/logger.ts

```diff
@@ -128,7 +134,7 @@ class InternalLogger {
 export const internalLogger: InternalLogger = new InternalLogger();
 
 /**
- * Set custom logger. Use the output function if it's set. Priority is higher than setLogFunction.
+ * Set custom logger. Use the output function if it's set. Priority is lower than setLogFunction.
```

Comment:
Add instructions that people also need to set log level?

Reply:
Updated example and README.